### PR TITLE
Add configurable small pages option

### PR DESCRIPTION
**Pull request by ffactory-ofcl**
Edit:
Now with even more magic and cross-browser compatibility! Pages are always A4 but when the small pages config is set and the room has a seatCount, two Din A5 pages get rotated and put on a single A4 sheet. Also now supports two html templates, one for A4 and one for A5.

As you can see in the following image, this also allows printing all QR codes at once without worrying about separate orientations, paper sizes or other things. The relevant locations just rotate their pages.
image

Old:
~Looks like this when printing in Chrome.~
image

~Firefox doesn't recognize the rotation automatically though, so the user has to select "landscape" before printing in this mode.~
When the config is 0 (default), nothing changes.

### CODE REVIEW

**Comment by functionaldude** [comment]
current behaviour when printing
image

**Comment by ffactory-ofcl** [comment]
@functionaldude your comment is now outdated, please adapt it ;)

**Comment by vonox7** [comment]
@ffactory-ofcl you can also add this feature to the features-list in Readme.md

**Review by vonox7 — APPROVED** [review]
Very nice! Did you also test what happens on "print all qr codes" if a room has an odd count of seats and then the location after it has no seats defined? so will then be the last page from the first room on the left side the last room and on the right side a blank A5 page?

File: src/serverMain/resources/viewQR/styles.css [hunk 1]
```diff
@@ -27,16 +27,38 @@ header > h2 {
   font-size: 32px;
 }
 
+.small-pages-container {
+  /* width must be 1mm less than A4 because of rounding errors that would otherwise leave empty pages */
+  height: 297mm;
+  width: 209mm;
+  page-break-after: always;
+  transform: translateX(209mm) rotate(90deg);
```
Comment (vonox7) [inline src/serverMain/resources/viewQR/styles.css:35]: btw, very nicely done in css :)

**Review by vonox7 — APPROVED** [review]

File: src/serverMain/resources/viewQR/styles.css (hunk 1 above)
Comment (ffactory-ofcl) [reply src/serverMain/resources/viewQR/styles.css:35]: Thanks, I think 80% of the time I spent wrestling css weirdness ^^

**Comment by ffactory-ofcl** [comment]
> @ffactory-ofcl you can also add this feature to the features-list in Readme.md

Will do

**Review by functionaldude — APPROVED** [review]

**Pull request merged**